src/app/createlog/page.tsx

```diff
@@ -19,18 +19,27 @@ interface Set {
   weight: number;
 }
 
+const convertWeight = (weight: number, unit: string) => {
+  const conversionRate = 2.20462;
+  if (unit === "kg") {
+    return weight / conversionRate;
+  } else {
+    return weight * conversionRate;
+  }
+};
+
 export default function CreateLog() {
   const { status, session } = useAuthSession(); // status, session and update are available, see auth-context.tsx
   const router = useRouter();
 
   const [searchInput, setSearchInput] = useState<string>("");
-  const [searchResults, setSearchResults] = useState<any>([]);
-  console.log(searchResults);
+  const [searchResults, setSearchResults] = useState<any[]>([]);
```

Comment:
TODO: I think we should figure out what the type the searchResults will actually be instead of using `any`